lib/prometheus/business_collector.rb

```diff
@@ -56,6 +56,12 @@ def initialize
     @client_systems_by_os = PrometheusExporter::Metric::Gauge.new(
       'client_systems_by_os', 'Systems by OS version (excluded Red Hat)'
     )
+    @total_reports_per_month = PrometheusExporter::Metric::Counter.new(
```

Comment:
TestResults (aka Reports) are deletable by users. A Gauge might be more appropriate (?).